docs/input-logstash.asciidoc

```diff
@@ -235,34 +231,11 @@ A path to a JKS- or PKCS12-formatted keystore with which to identify this plugin
 
 * Value type is <<password,password>>
 * There is no default value for this setting.
-* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_certificate>>
-* {prohibit-ssl-disabled-effective}
+* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_certificate>>.
+* Cannot be combined with configurations that disable SSL.
 
 A password or passphrase of the <<plugins-{type}s-{plugin}-ssl_key>>.
 
-[id="plugins-{type}s-{plugin}-ssl_truststore_path"]
```

Comment:
review note: removed `ssl_truststore_path` and `ssl_truststore_password` since we don't set them in `ssl_trust_options`. I will review and address them with SSL standardization support work.